solutions/cses-1745.mdx

```diff
@@ -2,20 +2,80 @@
 id: cses-1745
 source: CSES
 title: Money Sums
-author: Sofia Yang
+author: Sofia Yang, Qian Qian
 ---
 
 [Official Editorial](https://codeforces.com/blog/entry/70018)
 
+## Solution
+
+**Time Complexity:** $\mathcal{O}(N\cdot X)$
+
+## Implementation
+
 <LanguageSection>
 
-<JavaSection>
+<CPPSection>
 
-## Solution
+``` cpp
+#include <bits/stdc++.h>
 
-**Time Complexity:** $\mathcal{O}(N\cdot X)$
+using namespace std;
+
+const int MAX_N = 100;
+const int MAX_SUM = 1e5;
+
+// DP[i][j] is true, if it is possible to make a sum of j with i coins.
+bool dp[MAX_N + 1][MAX_SUM + 1];
+
+int main() {
+	int n;
+	cin >> n;
 
-## Java Implementation
+	vector<int> x(n);
+	for (int i = 0; i < n; i++) {
+		cin >> x[i];
+	}
+
+	dp[0][0] = true;
+
+	/**
+	 * Loop through all coins and all possible sums.
```

Comment:
i think its better off having this in the explanation rather than rows of comments

Reply:
I move it to the explanation part, but I didn't remove it from the Java version (which has the same comments), should I do that?

Reply:
yeah